test/MapRateTableTest.java

```diff
@@ -0,0 +1,78 @@
+import static org.junit.Assert.assertEquals;
+
+import java.math.BigDecimal;
+
+import org.junit.Test;
+
+/**
+ * Customized JUnit test fixture for {@code MapRateTable}.
+ */
+public class MapRateTableTest extends RateTableTest {
+
+    /**
+     * Test for populateRatesFromAPI.
+     */
+    @Test
+    public void testAPIPull1() {
+        RateTable t = new MapRateTable();
+        t.populateRatesFromAPI();
+        assertEquals(162, t.size());
+    }
+
```

Comment:
I'd like to see tests for clear, newInstance, transferFrom and the common methods (equals, toString, etc.).